### PR TITLE
Qa

### PR DESCRIPTION
**Pull request by tufranz**
Two of three points are done. Installed version is `2.5.13`.
To fix the version to `2.4.44` is the next point on my todo-list.

### CODE REVIEW

**Review by mb-wali — APPROVED** [review]

**Comment by mb-wali** [comment]
@tufranz  - can we control which version it will install, since every time it will properly install latest.

File: roles/de-slapd/tasks/main.yml [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
+    - installation
 
+# TODO: change to new configuration-format
 - name: "Generate the OpenLDAP Configuration File"
-  become: true
-  template:
+  ansible.builtin.template:
     src: "slapd.conf.j2"
-    dest: "/etc/openldap/slapd.conf"
+    dest: "/etc/ldap/slapd.conf"
```
Comment (mb-wali) [inline roles/de-slapd/tasks/main.yml:40]: is this directory name change is because of debian - or its not at all openldap you are trying to install?

Comment (tufranz) [reply roles/de-slapd/tasks/main.yml:40]: `/etc/ldap/` is the default config_dir in debians openldap-package.

**Comment by tufranz** [comment]
"@tufranz - can we control which version it will install, since every time it will properly install latest."
Very little. Using the distribution-package gives us little influence on the program-version. (I was not able do compile the Debian-source package for version `2.4.44`.)
OpenLdap only provides  source code. Compilation and installation is ready. I still work on startup and configuration.

**Pull request merged**